src/glue/jobs/s3_to_json.py

```diff
@@ -0,0 +1,209 @@
+"""
+S3 to JSON job
+
+Bucket/sort JSON from a zip archive provided by Care Evolution into
+JSON datasets in S3. A JSON dataset is specific to a key prefix
+(for example: json/dataset=EnrolledParticipants/) and only contains
+files which share a similar schema.
+"""
+import datetime
```

Comment:
Nit: Please follow the best practices for sorting imports: https://peps.python.org/pep-0008/#imports. There is vscode module that you can use that automatically does this as well.

https://marketplace.visualstudio.com/items?itemName=ms-python.isort.

By default you can also use the black vscode module to auto lint and structure your code.

Reply:
I don't use vscode. But if you know any vim plugins that do the same thing feel free to share.

Reply:
Highly recommend the switch to vscode, but here are some vim plugins I found by doing a quick google search.
* https://vimawesome.com/plugin/vim-isort
* https://vimawesome.com/plugin/black